recipes/Getting_Started_with_Jupyter_Locally/Getting_Started_with_Jupyter_Locally.ipynb

```diff
@@ -0,0 +1,154 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Running the Granite Code Recipe Notebooks Locally\n",
+    "How to run the Granite Code recipe Jupyter notebooks on your computer."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Clone the Granite Code Cookbook Repository\n",
+    "\n",
+    "Clone the repo and cd into the repo directory."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "shellscript"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "git clone git@github.com:ibm-granite-cookbooks/granite-code-cookbook.git\n",
```

Comment:
In addition to my other comment, all the shell commands have to start with `!`, e.g., `!git clone ...`. It appears that Jupyter allows you to do `pip install ...` without `!`, but when Ir un the other "shell cells", they fail unless I add `!`.

Reply:
I'll convert to a Markdown file, but noted for shell cells.